src/otx/recipe/_base_/data/mmseg_base.yaml

```diff
@@ -11,68 +11,70 @@ config:
     subset_name: train
     batch_size: 8
     num_workers: 4
-    transform_lib_type: MMSEG
+    transform_lib_type: TORCHVISION
     transforms:
-      - type: LoadImageFromFile
-      - reduce_zero_label: true
-        type: LoadAnnotations
-      - keep_ratio: false
-        ratio_range:
-          - 0.5
-          - 2.0
-        scale:
-          - 544
-          - 544
-        type: RandomResize
-      - cat_max_ratio: 0.75
-        crop_size:
-          - 512
-          - 512
-        type: RandomCrop
-      - prob: 0.5
-        type: RandomFlip
-      - type: PhotoMetricDistortion
-      - type: Pad
-        size:
-          - 512
-          - 512
-        pad_val:
-          img: 0
-          seg: 255
-      - type: PackSegInputs
+      - class_path: torchvision.transforms.v2.RandomResizedCrop
```

Comment:
It is a known issue that using torchvision operations shows slow performance.
What about replacing them with mmdet transforms implemented in torchvision pipeline like detection?

Reply:
Do we have it implemented in OTX? I will take a look

Reply:
https://github.com/openvinotoolkit/training_extensions/blob/3d6bbdf673c1d1072dc9813c48cdcc3a241c976d/src/otx/core/data/transform_libs/torchvision.py#L549C7-L549C24
However, this approach still has the issue of being slow. See also https://github.com/openvinotoolkit/training_extensions/issues/3379

Reply:
Yeah, conversion parts in the pipeline will be fixed or the whole pipeline can be updated by using other transform library.
But for now, please refer previously implemented ones.

Reply:
We don't support masks here unfortunately

Reply:
I added support for masks resizing, but I don't see any difference in speed (train/data_time and iter_time is the same during training), but I see drop in accuracy and worse convergence. Let's decide on our augmentations backend in next PRs. For now, I would like to leave torchvisionv2 for segmentation.

Reply:
It is fine if there is little difference between mmdet and torcivision on semantic segmentation.
Let's discuss the future plan later as you said.